src/WebJobs.Script/Diagnostics/Extensions/ExtensionBundleLoggerExtension.cs

```diff
@@ -25,31 +25,31 @@ internal static class ExtensionBundleLoggerExtension
 
         private static readonly Action<ILogger, string, string, Exception> _locateExtensionBundle =
             LoggerMessage.Define<string, string>(
-            LogLevel.Information,
+            LogLevel.Debug,
             new EventId(102, nameof(LocateExtensionBundle)),
```

Comment:
It is important that this message to appear in core tools, We have used this to identify issues root cause in a number of GitHub issues.

Reply:
Default log level when running in Core Tools is Info. Setting log level needs to be set to debug to see these logs.

Reply:
I don't think we can make all the extension bundle logs are debug. Would have to look at them individually.

Reply:
Ok. Reverted changes. Was trying to clean up info logs to debug :)

Reply:
I can review and mark extension bundle logs as debug. I think we only need a few of them to be info. Rest can be debug.